dubbo-remoting/dubbo-remoting-api/src/main/resources/META-INF/dubbo/internal/org.apache.dubbo.remoting.Dispatcher

```diff
@@ -1,4 +1,5 @@
 all=org.apache.dubbo.remoting.transport.dispatcher.all.AllDispatcher
+all2=org.apache.dubbo.remoting.transport.dispatcher.all2.AllDispatcher2

```

Comment:
Consider renaming "all2" to a significant symbol.

Reply:
This extension will move to dubbo-spi-extension in the next few month. I think we can rename it there.